Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -1,14 +1,23 @@
-import defined from "../../Core/defined.js";
-import ComponentDatatype from "../../Core/ComponentDatatype.js";
-import PrimitiveType from "../../Core/PrimitiveType.js";
 import AttributeType from "../AttributeType.js";
-import VertexAttributeSemantic from "../VertexAttributeSemantic.js";
+import BoundingSphere from "../../Core/BoundingSphere.js";
```

Comment:
@j9liu and I discussed this offline, we think it's possible to move these changes to its own pipeline stage that just generates the single attribute (and related uniforms/defines). This way we avoid bloating the geometry stage which is already one of the more complex stages.